openapi3/schema_test.go

```diff
@@ -161,6 +161,43 @@ var schemaExamples = []schemaExample{
 		},
 	},
 
+	{
+		Title: "NULLABLE ALLOF",
+		Schema: &Schema{
+			Type: "object",
+			Properties: map[string]*SchemaRef{
+				"value": NewAllOfSchema(
+					NewBoolSchema().WithNullable(),
+					NewBoolSchema().WithNullable(),
+				).NewRef()}},
+		Serialization: map[string]interface{}{
+			"type": "object",
+			"properties": map[string]interface{}{
+				"value": map[string]interface{}{
+					"allOf": []interface{}{
+						map[string]interface{}{
+							"nullable": true,
+							"type":     "boolean",
+						},
+						map[string]interface{}{
+							"nullable": true,
+							"type":     "boolean",
+						},
+					}}}},
+		AllValid: []interface{}{
+			nil,
+			true,
+			false,
+		},
+		AllInvalid: []interface{}{
+			2,
+			4.2,
+			[]interface{}{42},
+			"bla",
+			map[string]interface{}{},
+		},
+	},
```

Comment:
This test will always fail due to the schema being an object and expected valid values are not.

The following schema:

```json
{
	"type": "object",
	"properties": {
		"value": {
			"allOf": [
				  { "type": "bool", "nullable": true },
				  { "type": "bool", "nullable": true }
			]
		}
	}
}
```

Is valid with the following JSON:

```
{ "value": null }
{ "value": true }
{ "value": false }
```

But not with the following:

```
null
true
false
```